static_report/src/components/SingleReport/SingleReport.tsx

```diff
@@ -192,8 +220,9 @@ function TestsInformation({ data }) {
               <Th>Column</Th>
               <Th>Status</Th>
               <Th>Assertion</Th>
-              <Th>Expected</Th>
-              <Th>Actual</Th>
+              {type === 'piperider' && <Th>Expected</Th>}
```

Comment:
same comment here regarding separation of concerns (`dbt` vs `piperider` ) tables

Reply:
Agree with your opinions, but I think it's too early to separate tables. The dbt table is really simple currently, so I prefer not to change.